doc/tutorial.rst

```diff
@@ -0,0 +1,78 @@
+.. _tutorial:
+
+========
+Tutorial
+========
+
+**synced_collections** make it easy to store collections of data into various common data stores.
+The simplest backend to use is the JSON backend, which works out of the box using nothing more than the Python standard library.
+With this backend, a standard Python data collection may be automatically synchronized with a JSON file.
+
+.. code-block:: python
+
+    from synced_collections.backends.collection_json import JSONDict, JSONList
+
+    d = JSONDict("dict.json")
+    d["size"] = 10
+    d["color"] = "blue"
+
+    l = JSONList("list.json")
+    l = [10, "blue"]
+
+This will create the following JSON files:
+
+.. code-block:: bash
+
+    ~ $ cat dict.json
+    {'size': 10, 'color': 'blue'}
```

Comment:
This isn’t valid JSON. JSON requires double quotes…

Reply:
Not sure how this happened since I definitely thought I copied this straight from my terminal...